### ISSUE TITLE
Align dropdown design and functionality

### ISSUE THREAD

**Issue by sifferhans** [bcc-code/bmm-web#326]
We should probably make sure all dropdowns are consistent in functionality and design.
Eg. the track dropdown has a dark background on hover and keyboard navigation doesn't work that well.

https://github.com/bcc-code/bmm-web/assets/18753964/03f2baac-6fb2-4892-9bf1-b332b72f4414

**Comment by sifferhans** [comment]
@kkuepper Sure, I can create a base dropdown component we can use :)

**Comment by sifferhans** [comment]
@kkuepper Sorry, didn't finish this. Is this still relevant?

**Comment by kkuepper** [comment]
@sifferhans I was pretty sure it's still an issue. But it's actually fixed and I have no idea when that happened.
